docs/get-started/python.md

```diff
@@ -0,0 +1,127 @@
+                       spec=k8s.ServiceSpec(
+                           selector={"app": app_label},
+                           ports=[k8s.ServicePort(
+                               port=80, target_port=k8s.IntOrString.from_number(8080))],
+                           type="LoadBalancer"
+                       )
+                    )
+
+
+app = App()
+AwesomeApp(app, "getting-started", app_label="my-app")
+
+app.synth()
+```
+
+A few things worth noting about this sample:
+
+- An `AwesomeApp` class is created by passing the app instance, a string identifier "getting-started", and a custom `app_label` ("my-app") as arguments. This action generates and registers the Kubernetes resources defined in the `AwesomeApp` class within the application. We use the custom `app_label` parameter to dynamically set the "app" key in the "label" selectors and "metadata" labels for our Kubernetes resources.
+- The `__init__` method of the `AwesomeApp` class creates a Kubernetes [Service](https://kubernetes.io/docs/concepts/services-networking/service/) by instantiating the `k8s.KubeService` class with the following parameters: `self` as the scope of the Service resource within the `AwesomeApp` class, a string identifier "my-service", and a `k8s.ServiceSpec` object that defines the Service specification with a selector dictionary having the key "app" and the value set to the `app_label` parameter, a list of ports containing a single `k8s.ServicePort` object with `port` set to 80 and `target_port` set to 8080, and a `type` parameter with the value "LoadBalancer". The `app_label` parameter is passed to the `AwesomeApp` class.
+
+## Generating Kubernetes manifests
+After you have defined the Kubernetes resources for your project, you are ready to generate the Kubernetes manifest that will define your Service resource. 
+
+### Run the synth command
+1. Open a terminal and navigate to your project directory.
+2. Run the [synth](https://cdk8s.io/docs/latest/cli/synth/) command. This command generates a Kubernetes manifest file in the `dist` folder of your project directory. The manifest file is based on the parameters specified in your application.
+```console
+cdk8s synth
+```
+
+### View the manifest
+1. Open the `dist/getting-started.k8s.yaml` file. You should see a Kubernetes manifest similar to the following:
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: getting-started-my-service-c84dc095
+spec:
+  ports:
+    - port: 80
+      targetPort: 8080
+  selector:
+    app: my-app
+  type: LoadBalancer
+```
+
+## Wrap up
+Keep in mind that this is just a basic demonstration of CDK8s.
+- To continue your journey, develop a complete Kubernetes [Deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) and [Service](https://kubernetes.io/docs/concepts/services-networking/service/) using the [CDK8s Core](https://github.com/cdk8s-team/cdk8s-examples/blob/main/python/cdk8s-core/main.py) sample application.
```

Comment:
This is a bit confusing because it kind of alludes to another guide, but then just links to some code. Is this intentional or a place holder for something?